pages/Documents/RichMessaging/ViberTemplates/carousel-template.md

```diff
@@ -0,0 +1,351 @@
+---
+pagename: Carousel Template
+redirect_from:
+- structured-content-viber-templates-carousel-template.html
+  Keywords: structured content rich messaging
+  sitesection: Documents
+  categoryname: "Rich Messaging"
+  documentname: Viber Templates
+  permalink: viber-templates-carousel-template.html
+  indicator: messaging
+---
+
+### Overview
+
+The Viber carousel is a horizontally scrollable carousel of up to 10 vertical rich cards. Each card can include the following elements:
+
+* Title text
+* Subtitle text
+* Image (which will be placed above the title, subtitle and buttons) - image is not required to be placed in a card
+* Buttons with the following actions:
+    * Link
+    * Publish text
+
+Note:  When building a Viber carousel card you must keep in mind that each card consists of a total of 7 rows. Each row is an element
```

Comment:
@joshuajott can we catch that on the connector and reject the message if there is a card with more than 7 rows?